### PR TITLE
feat: appBridge-cli include appBridgeTheme in commands

### PR DESCRIPTION
**Pull request by anxobotana**
CU-8694vu23v

### CODE REVIEW

**Review by julianiff — APPROVED** [review]

File: packages/cli/src/commands/serve.ts [hunk 1]
```diff
@@ -66,6 +67,81 @@ class PlatformAppDevelopmentServer {
     }
 }
 
+class ThemeDevelopmentServer {
```
Comment (ragi96) [inline packages/cli/src/commands/serve.ts:70]: Maybe we should split these `DevelopmentServer` classes out of the `serve.ts` so we can add proper Ownership on them

File: packages/cli/src/servers/blockDevelopmentServer.ts
Comment (ragi96) [inline packages/cli/src/servers/blockDevelopmentServer.ts:1]: Can you add Ownership to each of these Server to the respective teams, so you can adjust them without doing too much noise

Comment (anxobotana) [reply packages/cli/src/servers/blockDevelopmentServer.ts:1]: there you go :)

**Review by ragi96 — APPROVED** [review]

File: packages/cli/src/servers/blockDevelopmentServer.ts
Comment (ragi96) [reply packages/cli/src/servers/blockDevelopmentServer.ts:1]: Thanks 💯

File: packages/cli/tests/utils/compiler.spec.ts [hunk 2]
```diff
@@ -33,13 +33,23 @@ describe('Compiler utils', () => {
...
             await import(outputFile);
+
             expect(global.window).toHaveProperty('index');
             expect(global.window?.index.block).toBe('this is a block');
             expect(global.window?.index.settings).toMatchObject({ some: 'settings' });
             expect(global.window?.index.dependencies['@frontify/app-bridge']).toBe('^3.0.0-beta.99');
         });
     });
 
+    describe('compile Theme', () => {
```
Comment (mike85) [inline packages/cli/tests/utils/compiler.spec.ts:44]: IMO we should also test whether the dependencies and settings are attached to the window. Wdyt?

**Review by mike85 — CHANGES_REQUESTED** [review]

File: packages/cli/tests/utils/compiler.spec.ts (hunk 2 above)
Comment (anxobotana) [reply packages/cli/tests/utils/compiler.spec.ts:44]: done

File: packages/cli/tests/files/compile-theme-files/package.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "test-theme",
+    "version": "1.0.0",
+    "main": "src/index.ts",
+    "dependencies": {
+        "@frontify/app-bridge": "^3.0.0-beta.99",
```
Comment (mike85) [inline packages/cli/tests/files/compile-theme-files/package.json:6]: Do we still need this?

**Review by mike85 — APPROVED** [review]

File: packages/cli/tests/files/compile-theme-files/package.json (hunk 3 above)
Comment (anxobotana) [reply packages/cli/tests/files/compile-theme-files/package.json]: not really, deleted

**Pull request merged**